### PR TITLE
Optimize site for desktop screens

### PR DESCRIPTION
**Pull request by BryteLitty**
I have made the following changes to the project:
- optimized the website for desktop using media queries
- arranged elements on the page using CSS flexbox and Grid

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -16,279 +16,524 @@ body {
...
+  row-gap: 88px;
+  background-color: #fbfbfb;
+  padding: 114px 0;
+}
+
+#works .card {
+  height: 552px;
+  padding: 16px;
+
+  /* gap: 12px; */
```
Comment (emmiiorji) [inline style.css]: Kindly endeavour to remove the commented styles in order to make the codebase more concise.

File: style.css [hunk 2]
```diff
@@ -16,279 +16,524 @@ body {
...
-  .skills ul {
-    padding-top: 10px;
-    width: 75%;
-    list-style-type: none;
-    display: flex;
-    justify-content: space-between;
-    color: #6070ff;
+  .card .skills ul {
+    width: 48%;
+    margin: 25px 0;
```
Comment (emmiiorji) [inline style.css]: Kindly ensure that the margin between the project skills matches your chosen template design across screen sizes.

Your design (Screen size - 1286px):
image

Figma Design:
image

File: style.css [hunk 3]
```diff
@@ -16,279 +16,524 @@ body {
...
+  background-color: white;
+  padding: 8px;
+  display: flex;
+  text-align: center;
+  color: #3965f2;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  border-radius: 8px;
+}
```
Comment (emmiiorji) [inline style.css:167]: You have done greatly at implementing a hover button interaction. Kindly implement all the button interactions to follow your chosen design template. 

Your design:
Enabled image hover image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @BryteLitty, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Almost there


## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow
- Design looks great


## Required Changes ♻️

- Kindly implement this curve that at the bottom left of the hero section of the page to match your chosen template
  Your design:
image

  Figma design:
image


_Check the comments under the review for more required changes._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BryteLitty** [comment]
I have updated the button interactions

File: style.css [hunk 4]
```diff
@@ -16,279 +16,529 @@ body {
...
   }
 
   .headline .container {
-    margin: 0 20px;
-    height: 587.15px;
-    padding-top: 115px;
-    display: flex;
-    flex-direction: column;
+    margin: 0 155px;
+    padding: 70px 70px;
```
Comment (BertrandConxy) [inline style.css]: - [ ] Good job so far 👏. As requested by the previous reviewer, please add the curving edge at the bottom left side of this headline section. Tools like [Clippy](https://bennettfeely.com/clippy/) can help you very much. Please try it out.

**your design**
image

**Figma design**
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @BryteLitty, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- No linter errors
- correct flow is used
- good improvement based on the previous reviewer's suggestions
- good PR


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BryteLitty** [comment]
Hi @BertrandConxy  please I am unable to use Clippy, the tool you recommended I use to achieve the curving edge at the bottom left side of this headline section. Can you please please give me directions on how to use it?

**Comment by BertrandConxy** [comment]
Hello @BryteLitty. If the Clippy tool is challenging to use, I would suggest that you use the `border-radius` technique. Here  I have found a good [YouTube tutorial](https://youtu.be/qUusNDtAOeA) that will help you get it done 😎. Try it out and you will inform me 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @BryteLitty 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
- [ ] Good job on implementing most of the requested changes 👍 Kindly add the `border-radius` for the headline section as requested.

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @BryteLitty  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**